kolibri/plugins/coach/assets/src/views/exams/exam-report-page/ManageExamModals.vue

```diff
@@ -100,12 +101,53 @@
           this.deactivateExam(this.exam.id);
         }
       },
+      handleUpdateExamDetails(details) {
+        const payload = {};
+        const assignmentsAreSame =
```

Comment:
could you use something like `lodash/xor` to see if the symmetric difference of the two assignments arrays differ in any way?